adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -34,24 +34,43 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
+class WiFiConnType: # pylint: disable=too-few-public-methods
+    """An enum-like class representing the different types of WiFi connections
+    that can be made. The values can be referenced like ``WiFiConnType.normal``.
+    Possible values are
+    - ``ThermocoupleType.normal``
+    - ``ThermocoupleType.enterprise``
+    """
+    # pylint: disable=invalid-name
+    normal = 1
+    enterprise = 2
+
 class ESPSPI_WiFiManager:
     """
     A class to help manage the Wifi connection
     """
-    def __init__(self, esp, secrets, status_pixel=None, attempts=2):
+    def __init__(self, esp, secrets, status_pixel=None, attempts=2, wificonntype=WiFiConnType.normal):
         """
         :param ESP_SPIcontrol esp: The ESP object we are using
         :param dict secrets: The WiFi and Adafruit IO secrets dict (See examples)
         :param status_pixel: (Optional) The pixel device - A NeoPixel or DotStar (default=None)
         :type status_pixel: NeoPixel or DotStar
         :param int attempts: (Optional) Failed attempts before resetting the ESP32 (default=2)
+        :param const con_type: (Optional) Type of WiFi connection: normal=1, WPA2 Enterprise=2
+        :param ~adafruit_esp32spi_wifimanager.WiFiConnType wificonntype: The type of WiFi \
+        connection to make. The default is "normal".
         """
         # Read the settings
         self._esp = esp
         self.debug = False
         self.ssid = secrets['ssid']
         self.password = secrets['password']
+        self.ent_ssid = secrets['ent_ssid']
+        self.ent_ident = secrets['ent_ident']
+        self.ent_user = secrets['ent_user']
+        self.ent_passwd = secrets['ent_passwd']
```

Comment:
Change this to `secrets['ent_password']` to keep convention with `secrets['password']`

Reply:
Agreed. Fixed in an upcoming commit.